op-deployer/book/src/reference-guide/releases.md

```diff
@@ -22,11 +22,24 @@ From time to time, we may backport bugfixes from develop onto earlier versions o
 as follows:
 
 1. If one doesn't exist already, make a new branch for the version lineage you're patching (e.g. `v0.2.x`). This branch
-   should be based on the latest release of that lineage. The branch should be named as follows: 
+   should be protected (not deletable) and should be based on the latest release of that lineage. The branch should be named as follows:
    `backports/op-deployer/<lineage, i.e. v0.2.0>`.
 2. Open a PR with the backport against that branch. Be sure to reference the original commit in the backport.
 3. Make and push a new tag on that lineage.
 
+Example for backporting fix(es) from `develop` and created a new release `op-deployer/v0.2.1`:
+```
+git checkout -b backports/op-deployer/v0.2.0 op-deployer/v0.2.0
+git push origin backports/op-deployer/v0.2.0
+git checkout -b op-deployer/v0.2.1 backports/op-deployer/v0.2.0
+git cherry-pick <commit-hash>
+git push origin op-deployer/v0.2.1
```

Comment:
For the sake of this example, can we use a more informal name instead `op-deployer/v0.2.1` so that its representative of the fleeting nature of the source branch in the backport PR?

Had me confused for a sec seeing it named the same as the to-be release name "op-deployer/v0.2.1"